### PR TITLE
Adding stalebot

### PR DESCRIPTION
**Pull request by dry923**
Adding configuration file for stalebot - https://github.com/probot/stale

### CODE REVIEW

File: .github/stale.yml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# Number of days of inactivity before an Issue or Pull Request becomes stale
+daysUntilStale: 15
+
+# Number of days of inactivity before an Issue or Pull Request with the stale label is closed.
+# Set to false to disable. If disabled, issues still need to be closed manually, but will remain marked as stale.
+daysUntilClose: 10
```
Comment (aakarshg) [inline .github/stale.yml]: I'm not entirely sure that it's a good idea to auto close PRs/Issues in 10 days.

Comment (dry923) [reply .github/stale.yml]: It's 10 days after they get marked as stale. So it would be 25 days total from last touch. Would 14 make you happier?

Comment (jtaleric) [reply .github/stale.yml]: I think 10 is MORE than enough time personally. Is there a way for admins to override?

Comment (dry923) [reply .github/stale.yml]: There's an option to ignore certain labels. We could use that as a way to override.

**Pull request merged**